### PR TITLE
Se sube entregable3

### PR DESCRIPTION
**Pull request by ecedillocs**

### CODE REVIEW

File: Eduardo_Antonio_Cedillo_Segovia/Entregable3/LoadBalancer/src/main/java/com/codeusingjava/LoadBalancerApplication.java [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+import org.springframework.cloud.client.loadbalancer.LoadBalanced;
+import org.springframework.cloud.netflix.ribbon.RibbonClient;
+import org.springframework.context.annotation.Bean;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.client.RestTemplate;
+
+import com.codeusingjava.config.RibbonConfiguration;
+
+@SpringBootApplication(scanBasePackages= {
```
Comment (Wizeline-GerardoRuiz) [inline Eduardo_Antonio_Cedillo_Segovia/Entregable3/LoadBalancer/src/main/java/com/codeusingjava/LoadBalancerApplication.java]: Esta aplicación (LoadBalancer) no debe ser parte de la entrega final, hay que eliminarla para considerarla válida

File: Eduardo_Antonio_Cedillo_Segovia/Entregable3/learningjavamaven/DockerFile [hunk 2]
```diff
@@ -0,0 +1,5 @@
+FROM openjdk:11
```
Comment (Wizeline-GerardoRuiz) [inline Eduardo_Antonio_Cedillo_Segovia/Entregable3/learningjavamaven/DockerFile:1]: El nombre del archivo está mal, hay que corregirlo

File: Eduardo_Antonio_Cedillo_Segovia/Entregable3/learningjavamaven/DockerFile.txt [hunk 3]
```diff
@@ -0,0 +1,5 @@
+FROM openjdk:11
```
Comment (Wizeline-GerardoRuiz) [inline Eduardo_Antonio_Cedillo_Segovia/Entregable3/learningjavamaven/DockerFile.txt]: Este archivo es inválido, hay que eliminarlo

File: Eduardo_Antonio_Cedillo_Segovia/Entregable3/learningjavamaven/src/test/java/com/wizeline/maven/learningjavamaven/controller/UserControllerTest.java [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+    public static final String USER = "Admin";
+    public static final String CODIGO_EXITO = "OK";
+    private static final String RESPUESTA = "respuesta";
+
+
+    @Test
+    public void loginTest() {
+        String user = "usuario";
+        String password = "123456";
+        ResponseDTO response = new ResponseDTO();
```
Comment (Wizeline-GerardoRuiz) [inline Eduardo_Antonio_Cedillo_Segovia/Entregable3/learningjavamaven/src/test/java/com/wizeline/maven/learningjavamaven/controller/UserControllerTest.java]: Hay que incluir mensajes en la consola usando Logger para indicar los pasos del ciclo de vida del caso de prueba

File: Eduardo_Antonio_Cedillo_Segovia/Entregable3/learningjavamaven/src/test/java/com/wizeline/maven/learningjavamaven/controller/UserControllerTest.java [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+import org.springframework.http.HttpStatus;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.mockito.ArgumentMatchers.anyString;
+import static org.mockito.Mockito.when;
+
+import javax.servlet.http.HttpSession;
+
+@ExtendWith(MockitoExtension.class)
+public class UserControllerTest {
```
Comment (Wizeline-GerardoRuiz) [inline Eduardo_Antonio_Cedillo_Segovia/Entregable3/learningjavamaven/src/test/java/com/wizeline/maven/learningjavamaven/controller/UserControllerTest.java:34]: Hay que incluir más pruebas para el Happy Path y pruebas que contemplen Edge Cases de tu aplicación

File: Eduardo_Antonio_Cedillo_Segovia/Entregable3/learningjavamaven/pom.xml [hunk 6]
```diff
@@ -0,0 +1,182 @@
...
+			<version>2.3.0-b170127.1453</version>
+		</dependency>
+		<dependency>
+			<groupId>com.github.vladimir-bukhtoyarov</groupId>
+			<artifactId>bucket4j-core</artifactId>
+			<version>7.6.0</version>
+		</dependency>
+	</dependencies>
+	<build>
+		<plugins>
```
Comment (Wizeline-GerardoRuiz) [inline Eduardo_Antonio_Cedillo_Segovia/Entregable3/learningjavamaven/pom.xml:171]: Hay que incluir el plugin de JaCoCo o de SonarCloud y configurarlos para considerarlo completado

**Review by Wizeline-GerardoRuiz — CHANGES_REQUESTED** [review]

File: Eduardo_Antonio_Cedillo_Segovia/Entregable3/LoadBalancer/src/main/java/com/codeusingjava/LoadBalancerApplication.java (hunk 1 above)
Comment (ecedillocs) [reply Eduardo_Antonio_Cedillo_Segovia/Entregable3/LoadBalancer/src/main/java/com/codeusingjava/LoadBalancerApplication.java]: Ya se elimino en la ultima entrega

File: Eduardo_Antonio_Cedillo_Segovia/Entregable3/learningjavamaven/DockerFile (hunk 2 above)
Comment (ecedillocs) [reply Eduardo_Antonio_Cedillo_Segovia/Entregable3/learningjavamaven/DockerFile:1]: Se corrigio el nombre del archivo

File: Eduardo_Antonio_Cedillo_Segovia/Entregable3/learningjavamaven/DockerFile.txt (hunk 3 above)
Comment (ecedillocs) [reply Eduardo_Antonio_Cedillo_Segovia/Entregable3/learningjavamaven/DockerFile.txt]: El archivo se ha eliminado

File: Eduardo_Antonio_Cedillo_Segovia/Entregable3/learningjavamaven/src/test/java/com/wizeline/maven/learningjavamaven/controller/UserControllerTest.java (hunk 4 above)
Comment (ecedillocs) [reply Eduardo_Antonio_Cedillo_Segovia/Entregable3/learningjavamaven/src/test/java/com/wizeline/maven/learningjavamaven/controller/UserControllerTest.java]: Se incluyeron mensajes en la consola con Logger

**Comment by ecedillocs** [comment]
Se corrigieron los puntos mencionados. gracias

**Review by Wizeline-GerardoRuiz — APPROVED** [review]
LGTM 👍
